geode-core/src/main/java/org/apache/geode/distributed/internal/ShutdownMessage.java

```diff
@@ -79,7 +79,7 @@ protected void process(final ClusterDistributionManager dm) {
     // reply.setRecipient(getSender());
```

Comment:
looks like some dead code cruft ... maybe we scrape it off while we're nearby

Reply:
removed